arch/arm/boot/dts/zynq-enclustra-zx-common.dtsi

```diff
@@ -1,32 +1,32 @@
 / {
-	memory {
+	memory@20000000 {
```

Comment:
I think this should be memory@0

Reply:
You are absolutely right, sorry about that. Fix pushed.